homeassistant/components/androidtv/media_player.py

```diff
@@ -436,9 +466,13 @@ def adb_command(self, cmd):
 class AndroidTVDevice(ADBDevice):
     """Representation of an Android TV device."""
 
-    def __init__(self, aftv, name, apps, turn_on_command, turn_off_command):
+    def __init__(
+        self, aftv, name, apps, get_sources, turn_on_command, turn_off_command
+    ):
         """Initialize the Android TV device."""
-        super().__init__(aftv, name, apps, turn_on_command, turn_off_command)
+        super().__init__(
+            aftv, name, apps, get_sources, turn_on_command, turn_off_command
+        )
 
         self._device = None
```

Comment:
Correct, so I can remove it from the initialization call but it will still be created and set during  update calls. I will submit a PR to your repo to remove it from the update call entirely since it no longer serves a purpose

Reply:
Ah I see how you addressed that in your PR. Still think it's worth removing entirely unless you see value in the parameter for other use cases

Reply:
Actually, removing device looks to be a pretty significant change, and maybe it would provide value for people who want to use `python-androidtv` for other reasons. So my suggestion is that instead of removing it, it is renamed to `audio_device` to be more self descriptive and left as is. I will submit a PR with that proposed change for you to review